### PR TITLE
Logstash Keycloak Filter

### PR DESCRIPTION
**Pull request by r2o3k**

### CODE REVIEW

File: app/src/test/java/elk/plugins/KeycloakFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        KeycloakFilter filter = new KeycloakFilter("keycloak_filter", config, context);
+
+        Event e = new org.logstash.Event();
+        e.setField("authorization", "abcdef");
+        
+        TestMatchListener matchListener = new TestMatchListener();
+        Collection<Event> results = filter.filter(Collections.singletonList(e), matchListener);
+
+        assertEquals(1, results.size());
+        assertEquals("fedcba", e.getField("authorization"));
```
Comment (EandrewJones) [inline app/src/test/java/elk/plugins/KeycloakFilterTest.java]: Is there a reason the string is reversed?

**Review by EandrewJones — APPROVED** [review]
LGTM. This is great work. I left one small question about something that seemed odd.

File: app/src/test/java/elk/plugins/KeycloakFilterTest.java (hunk 1 above)
Comment (r2o3k) [reply app/src/test/java/elk/plugins/KeycloakFilterTest.java]: Oh, it was from the example. I stopped focusing on implementing the test and worked more on getting the plugin ready. I should go back and make a valid test.

Comment (EandrewJones) [reply app/src/test/java/elk/plugins/KeycloakFilterTest.java]: Yes please. At minimum, write a test proving a malicious actor can't just add `authorized = true` to the logs and bypass the filter.

Based on the logic, this shouldn't happen, but it's worth having a test for that.

File: app/src/test/java/elk/plugins/KeycloakFilterTest.java
Comment (EandrewJones) [inline app/src/test/java/elk/plugins/KeycloakFilterTest.java:1]: Thank you for the test.

**Review by EandrewJones — APPROVED** [review]

**Pull request merged**